### PR TITLE
Generate hot link design

### PR DESCRIPTION
**Pull request by gregory-latinier**
First step integration

### CODE REVIEW

**Comment by bonustrack** [comment]
- Small bug in step 2 there is a space after the Select operation input:
image

- On the last step the link to copy should be inside an input and take all red rectangle space:
image

- Would be nice if we can click "SELECT OPERATION" in the steps to go back

- We should use default description from the blockchain code, you must to search for the operation like this https://github.com/steemit/steem/blob/8cd5f688d75092298bcffaa48a543ed9b01447a6/libraries/wallet/include/steemit/wallet/wallet.hpp#L604 
Leave it empty if there is no description

File: .nsprc [hunk 1]
```diff
@@ -1,3 +1,6 @@
 {
-  "exceptions": ["https://nodesecurity.io/advisories/525"]
+  "exceptions": [
+    "https://nodesecurity.io/advisories/525",
+    "https://nodesecurity.io/advisories/527"
```
Comment (bonustrack) [inline .nsprc]: We should upgrade to Express version 4.15.5, it's an advice on nodesecurity.io  we should not get used to add exceptions.

Comment (bonustrack) [reply .nsprc]: Sorry actually you did update Express, so why the exeption? Isnt working?

Comment (gregory-latinier) [reply .nsprc]: The project has already been updated to 4.15.5 but there is still a security issue with this version

Comment (gregory-latinier) [reply .nsprc]: The last version of express doesn't use the upgraded version of a subpackage. Until they release a 4.15.6 I guess we have to disable this exception

Comment (bonustrack) [reply .nsprc]: Ok

**Comment by bonustrack** [comment]
Can we have the icon `>` centered verticaly? Also there should be a margin or padding around the icon, sort of text dont come to close.
image

**Comment by bonustrack** [comment]
Description text font-size should be `16px`

**Pull request merged**